frontend/pages/index.vue

```diff
@@ -59,132 +59,122 @@ export default Vue.extend({
   },
   data() {
     return {
-      homeSections: [] as HomeSection[],
       carouselItems: [] as BaseItemDto[]
     };
   },
-  async fetch() {
-    const validSections = ['resume', 'resumeaudio', 'upnext', 'latestmedia'];
-
-    // Filter for valid sections in Jellyfin Vue
-    // TODO: Implement custom section order
-    let homeSectionsArray = pickBy(
-      // @ts-expect-error - No typings for this
-      this.clientSettings.CustomPrefs,
-      (value: string, key: string) => {
-        return (
-          value &&
-          validSections.includes(value) &&
-          key.startsWith('homesection')
-        );
-      }
-    );
-
-    if (!Object.keys(homeSectionsArray).length) {
-      homeSectionsArray = {
-        homeSection0: 'librarytiles',
-        homeSection1: 'resume',
-        homeSection2: 'resumeaudio',
-        homeSection3: 'upnext',
-        homeSection4: 'latestmedia'
-      };
-    }
-
-    // Convert to an array
-    homeSectionsArray = Object.values(homeSectionsArray);
-
-    const homeSections: HomeSection[] = [];
-
-    for (const homeSection of homeSectionsArray as Array<string>) {
-      switch (homeSection) {
-        case 'librarytiles': {
-          homeSections.push({
-            name: 'libraries',
-            libraryId: '',
-            shape: CardShapes.Thumb,
-            type: 'libraries'
-          });
-          break;
+  head() {
+    return {
+      title: this.page.title
+    };
+  },
+  computed: {
+    ...mapStores(clientSettingsStore, pageStore, userViewsStore),
+    homeSections(): HomeSection[] {
+      const validSections = ['resume', 'resumeaudio', 'upnext', 'latestmedia'];
+
+      // Filter for valid sections in Jellyfin Vue
+      // TODO: Implement custom section order
+      let homeSectionsArray = pickBy(
+        // @ts-expect-error - No typings for this
+        this.clientSettings.CustomPrefs,
+        (value: string, key: string) => {
+          return (
+            value &&
+            validSections.includes(value) &&
+            key.startsWith('homesection')
+          );
         }
-        case 'latestmedia':
-          {
-            const latestMediaSections = [];
-
-            if (!this.userViews.views) {
-              await this.userViews.refreshUserViews();
-            }
+      );
+
+      if (!Object.keys(homeSectionsArray).length) {
+        homeSectionsArray = {
+          homeSection0: 'librarytiles',
+          homeSection1: 'resume',
+          homeSection2: 'resumeaudio',
+          homeSection3: 'upnext',
+          homeSection4: 'latestmedia'
+        };
+      }
 
-            const excludeViewTypes = [
-              'playlists',
-              'livetv',
-              'boxsets',
-              'channels'
-            ];
-
-            for (const userView of this.userViews.views) {
-              if (
-                excludeViewTypes.includes(userView.CollectionType as string)
-              ) {
-                continue;
+      // Convert to an array
```

Comment:
This comment is a little bit meaningless imo, as it's implicit in the next line. A better comment would be one that answers the question of: Why we need this converted to an array?

Reply:
I didn't put it in and I feel like it's unnecessary. So I think I should just let it here ^^

Reply:
Done